src/page_load.js

```diff
@@ -0,0 +1,196 @@
+        <a href="#" class="card-footer-item">Order</a>
+      </footer>
+    </div>
+  </div>
+  <div class="column">
+    <div class="card">
+      <div class="card-image">
+        <figure class="image is-3by2">
+          <img
+            src="https://www.acouplecooks.com/wp-content/uploads/2019/12/Vegan-Fettucini-Alfredo-005.jpg"
+            alt="Placeholder image"
+          />
+        </figure>
+      </div>
+      <div class="card-content">
+        <div class="content">
+          Lorem ipsum leo risus, porta ac consectetur ac, vestibulum at eros.
+          Donec id elit non mi porta gravida at eget metus. Cum sociis natoque
+          penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+          Cras mattis consectetur purus sit amet fermentum.
+        </div>
+      </div>
+      <footer class="card-footer">
+        <a href="#" class="card-footer-item">Save</a>
+        <a href="#" class="card-footer-item">Add</a>
+        <a href="#" class="card-footer-item">Order</a>
+      </footer>
+    </div>
+  </div>
+  <div class="column">
+    <div class="card">
+      <div class="card-image">
+        <figure class="image is-3by2">
+          <img
+            src="https://www.acouplecooks.com/wp-content/uploads/2019/12/Vegan-Fettucini-Alfredo-005.jpg"
+            alt="Placeholder image"
+          />
+        </figure>
+      </div>
+      <div class="card-content">
+        <div class="content">
+          Lorem ipsum leo risus, porta ac consectetur ac, vestibulum at eros.
+          Donec id elit non mi porta gravida at eget metus. Cum sociis natoque
+          penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+          Cras mattis consectetur purus sit amet fermentum.
+        </div>
+      </div>
+      <footer class="card-footer">
+        <a href="#" class="card-footer-item">Save</a>
+        <a href="#" class="card-footer-item">Add</a>
+        <a href="#" class="card-footer-item">Order</a>
+      </footer>
+    </div>
+  </div>
+  <div class="column">
+    <div class="card">
+      <div class="card-image">
+        <figure class="image is-3by2">
+          <img
+            src="https://www.acouplecooks.com/wp-content/uploads/2019/12/Vegan-Fettucini-Alfredo-005.jpg"
+            alt="Placeholder image"
+          />
+        </figure>
+      </div>
+      <div class="card-content">
+        <div class="content">
+          Lorem ipsum leo risus, porta ac consectetur ac, vestibulum at eros.
+          Donec id elit non mi porta gravida at eget metus. Cum sociis natoque
+          penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+          Cras mattis consectetur purus sit amet fermentum.
+        </div>
+      </div>
+      <footer class="card-footer">
+        <a href="#" class="card-footer-item">Save</a>
+        <a href="#" class="card-footer-item">Add</a>
+        <a href="#" class="card-footer-item">Order</a>
+      </footer>`;
```

Comment:
- [ ] [required] Please use `document.createElement` and `document.appendChild` to create and append HTML elements to the `<div>` with `id=content`. This is an opportunity for you to practice/explore building HTML elements using only javascript. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))